### ISSUE TITLE
Cache option not working

### ISSUE THREAD

**Issue by do-web** [storyblok/storyblok-nuxt#149]
If i try to enable cache option:
`
    buildModules: [
        ["@storyblok/nuxt", {
            accessToken: "xxxxx",
            apiOptions: {
                cache: { type: "memory" },
            },
        }]
    ],
`

I get an error:
Cannot add property accessToken, object is not extensible

This is also not working:

`
    buildModules: [
        ["@storyblok/nuxt", {
            accessToken: "xxxxx",
            apiOptions: {
                accessToken: "xxxxx",
                cache: { type: "memory" },
            },
        }]
    ],
`

Iam using the latest version.

If i remove the apiOptions property, storyblok creates two request:

`301 https://api.storyblok.com/v2/cdn/stories/en?version=published&token=xxxxx&cv=undefined` =>
`200 https://api.storyblok.com/v2/cdn/stories/en?cv=1654702667&token=xxxxx&version=published`

How can i prevent this 301?

**Comment by arpadgabor** [comment]
Confirming this issue. Whenever I pass `apiOptions` to the module it start failing with `[nuxt] [request error] Cannot add property accessToken, object is not extensible`.

The problematic code seems to be this bit in the `storyblokInit` function (code taken from dist folder in node_modules):
```
const { bridge, accessToken, use = [], apiOptions = {} } = pluginOptions;
apiOptions.accessToken = apiOptions.accessToken || accessToken;
```

**Comment by StaffOfHades** [comment]
This should be fixed as of `V4.3.0`. Check https://github.com/storyblok/storyblok-nuxt/issues/170#issuecomment-1239527769.

**Comment by alvarosabu** [comment]
Closing since it hasn't been active in a while and the latest comment from @StaffOfHades. If it's still happening feel free to re-open providing a valid reproduction link

Thanks!
